lib/generators/intercom_app/install/install_generator.rb

```diff
@@ -35,11 +40,20 @@ def create_intercom_session_repository_initializer
         copy_file 'intercom_session_repository.rb', 'config/initializers/intercom_session_repository.rb'
       end
 
-
       def mount_engine
         route "mount IntercomApp::Engine, at: '/'"
       end
 
+      private
+
+      def generate_signature
```

Comment:
it's not really clear what signature you're generating. you're not signing webhooks, you're receiving them